VideoEncoding/EncodingWithMESPredefinedPreset/appsettings.json

```diff
@@ -1,14 +1,14 @@
 {
-    "_comment": "For security reasons, please do not check in this file to source control.",
-    "AadClientId": "00000000-0000-0000-0000-000000000000",
-    "AadEndpoint": "https://login.microsoftonline.com",
-    "AadSecret": "00000000-0000-0000-0000-000000000000",
-    "AadTenantId": "00000000-0000-0000-0000-000000000000",
-    "AccountName": "amsaccount",
-    "ArmAadAudience": "https://management.core.windows.net/",
-    "ArmEndpoint": "https://management.azure.com/",
-    "Region": "West US 2",
-    "ResourceGroup": "amsResourceGroup",
-    "SubscriptionId": "00000000-0000-0000-0000-000000000000"
+  "_comment": "For security reasons, please do not check in this file to source control.",
+  "AadClientId": "3459069d-e768-4ee2-9183-17f517663919",
+  "AadEndpoint": "https://login.microsoftonline.com",
+  "AadSecret": "9557e512-3c05-46d6-b90d-e94ef15dcd10",
+  "AadTenantId": "72f988bf-86f1-41af-91ab-2d7cd011db47",
+  "AccountName": "xpouyatdemo",
```

Comment:
@xpouyat  - UH OH!  YOU BETTER BACK THIS ONE OUT!!!!

Reply:
ok, sure and I will renew the SP